doc/whats-new.rst

```diff
@@ -13,11 +13,35 @@ What's New
     import xarray as xr
     np.random.seed(123456)
 
-.. _whats-new.0.13.1:
+.. _whats-new.0.14.0:
 
-v0.13.1 (unreleased)
```

Comment:
hmmm.. if we're bumping major versions so soon, a bunch of deprecation warnings need to be fixed. Also we might consider releasing 0.13.1 so that the bugfixes for 0.13's  concat refactor and groupby deprecations can make it out.

Reply:
Slided all deprecations from 0.14 to 0.15.
Why shouldn't we put the bugfixes you mention directly in 0.14?